draft-johansson-ccwg-rfc8298bis-screamv2.md

```diff
@@ -828,7 +837,7 @@ if (is_ce_t)
     backoff_t = l4s_alpha / 2
 
     # Increase stability for very small ref_wnd
-    backOff_t *= max(0.8, 1.0 - ref_wnd_ratio * 2)
+    backOff_t *= max(0.5, 1.0 - ref_wnd_ratio)
```

Comment:
Appears to be algorithmic change, is this intended?

Reply:
Explained in new section Algorithm changes